src/nplinker/__init__.py

```diff
@@ -8,6 +9,11 @@
 __version__ = "2.0.0-alpha.1"
 
 
+# The path to the NPLinker application database directory
+NPLINKER_APP_DATA_DIR = Path(__file__).parent / "data"
+del Path
```

Comment:
Why is the deletion needed?

Reply:
otherwise the `Path` will be included in the attributes of `nplinker` module, which means you can use `nplinker.Path`.